### ISSUE TITLE
Not Working on Google Pages

### ISSUE THREAD

**Issue by viliampucik** [mozlima/redirectbypasser#4]
Only tested on Google pages which used to work in the past but not any more. Even if all other add-ons are disabled it still does not show the hover icons on Google, Google Mail nor Google Images pages.

**Comment by mozlima** [comment]
Hi,

In the option page > advanced is  "Check content inserted, after page load" and "Detect attribute changes" enabled?

**Comment by viliampucik** [comment]
Yes, it is enabled, but still not working :( When I click on a link in GMail email, it still forwards my browser to Google (https://www.google.com/url?q=https%3A%2F%2F...) first and then to the original URL.

**Comment by ChaosDMNS** [comment]
You need to right click on the link first... that will trigger the google script that changes the urls.

**Comment by tearwyx** [comment]
How to make option "Replace URL of the link" working after right click on Google Images?
